ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/TimestampScheduleBuilderTest.java

```diff
@@ -149,4 +149,22 @@ public void streamMilestoneBlocksReturnTimestampsInOrder() {
 
     assertThat(schedule.streamMilestoneBlocks()).containsExactly(FIRST_TIMESTAMP_FORK, 2L, 3L, 5L);
   }
+
+  @Test
+  public void isOnForkBoundary() {
```

Comment:
This doesn't seem like the right place for this test as it's not implemented on the builder. Suggest putting in a TimestampScheduleTest instead.

Reply:
I agree. Current test structure follows ProtocolScheduleBuilderTest which conflates tests for both the builder and class itself. 

Since this issue affects multiple Schedule methods on both schedule types, I would like to handle this all in one go as one PR though, can play straight after this.